<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="p1">
     <widget class="QLabel" name="Name">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reservation Name:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="rmType">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Room Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="rmChoice">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Room Choice:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="queen">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>231</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>2 Queen Room (Maximum Guests: 4)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="king">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>241</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>1 King Room (Maximum Guests: 3)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="atrium">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>171</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Atrium</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="standard">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>130</y>
        <width>171</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Standard</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>571</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="dates">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Dates of Stay:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lengthofstay">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Length of Stay (Nights):</string>
      </property>
     </widget>
     <widget class="QLabel" name="adults">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>230</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Adults: </string>
      </property>
     </widget>
     <widget class="QLabel" name="kids">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Children:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="beginDate">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="to">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>250</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="endDate">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="numNights">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="adultCt">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>250</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="childCt">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>300</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="estimateLbl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Estimated Cost (before Taxes &amp; Fees):</string>
       </property>
      </widget>
      <widget class="QLabel" name="cost_output">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>10</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>$ 0.00</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>370</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Page</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="p2">
     <widget class="QLabel" name="rmChoiceConfirm">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Room:</string>
      </property>
     </widget>
     <widget class="QLabel" name="tax">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Tax:</string>
      </property>
     </widget>
     <widget class="QLabel" name="parkFee">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking Fee:</string>
      </property>
     </widget>
     <widget class="QLabel" name="resortFee">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Resort Fee:</string>
      </property>
     </widget>
     <widget class="QFrame" name="totalDueFm">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>261</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QLabel" name="totalDue">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Due:</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="paymentMethod">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Payment Method:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="visaRadio">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>VISA</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="mastcrdRadio">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Mastercard</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="amExprssRadio">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>American Express</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="discoveryRadio">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Discovery</string>
      </property>
     </widget>
     <widget class="QLabel" name="cardNumLbl">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Card Number:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="cardNumInput">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pay">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>360</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pay</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="p3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReservation_Form">
    <property name="title">
     <string>Reservation Form</string>
    </property>
   </widget>
   <addaction name="menuReservation_Form"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
